bundles/azure/src/main/java/org/apache/gravitino/abs/fs/AzureFileSystemProvider.java

```diff
@@ -74,7 +74,7 @@ public FileSystem getFileSystem(@Nonnull Path path, @Nonnull Map<String, String>
 
     hadoopConfMap.forEach(configuration::set);
 
-    return FileSystem.get(path.toUri(), configuration);
+    return FileSystem.newInstance(path.toUri(), configuration);
```

Comment:
What's the difference?

Reply:
FileSystem has cache holds for FileSystem instances if the same URI is used with different configurations. this could lead to the problem that using credential and user static key share the same filesystem instance.